### PR TITLE
Add documentation on logging formats and escaping logs

### PR DESCRIPTION
**Pull request by nnshah1**

### CODE REVIEW

File: docs/protocol/extension_logging.md [hunk 1]
```diff
@@ -196,3 +212,74 @@ message LogSettingsResponse
...
+| simple | 1       | READY  |
++--------+---------+--------+
+```
+
+
+2. TRITONSERVER_LOG_ISO8601
+
+### Single-line Entry
+```
+<year>-<month>-<day>T<hour>:<min>:<sec>Z <level> <pid> <file>:<line>] <message>
```
Comment (whoisj) [inline docs/protocol/extension_logging.md:262]: same question as on core, does the format include `.<msec>`?

Comment (nnshah1) [reply docs/protocol/extension_logging.md:262]: it doesn't currently - that is a future enhancement though - we decided not to add it in these set of changes

File: docs/protocol/extension_logging.md [hunk 2]
```diff
@@ -117,8 +117,24 @@ $log_setting_request =
...
+- "log_error" : a `$boolean` parameter that controls whether the Triton server logs ERROR level messages.
+
+- "log_verbose_level" : a `$number` parameter that controls whether the Triton server outputs verbose messages
+of varying degrees. This value can be any integer >= 0. If "log_verbose_level" is 0, verbose logging will be disabled, and
+no verbose messages will be output by the Triton server. If "log_verbose_level" is 1, level 1 verbose messages will be output
+by the Triton server. If "log_verbose_level" is 2, the Triton server will output all verbose messages of
+level <= 2, etc. Attempting to set "log_verbose_level" to a number < 0 will result in an error.
+
+- "log_format" : a `$string` parameter that controls the format of Triton server log messages. There are currently
+2 formats: "default" and "ISO8601".
```
Comment (rmccorm4) [inline docs/protocol/extension_logging.md:137]: Can we avoid the dupe descriptions so there's one less place to update if something changes (ex: new log_format)? 
Maybe something like this?
```suggestion
settings (described above) can be updated:
- "log_info"
- "log_warning"
- "log_error"
- "log_verbose_level"
- "log_format" 
```

**Review by rmccorm4 — COMMENTED** [review]
LGTM other than comment about duplicating descriptions

File: docs/protocol/extension_logging.md [hunk 3]
```diff
@@ -117,8 +117,14 @@ $log_setting_request =
...
-When a `$log_setting` JSON is received (defined above), only the specified
-settings will be updated.
+When a `$log_setting` JSON is received (defined above), only the
+specified settings will be updated. Currently, the following log
+settings (described above) can be updated:
+- "log_info"
+- "log_warning"
+- "log_error"
+- "log_verbose_level"
+- "log_format"
```
Comment (rmccorm4) [inline docs/protocol/extension_logging.md:127]: Fix linting
```suggestion
- "log_info"
- "log_warning"
- "log_error"
- "log_verbose_level"
- "log_format"
```

**Review by rmccorm4 — APPROVED** [review]

**Pull request merged**